apps/mobile/components/ai-elements/file-diff-tool.tsx

```
@@ -0,0 +1,336 @@
+
+	return (
+		<View>
+			{visibleLines.map((line, i) => (
+				<View
+					className={cn(
+						"flex-row border-l-2 px-2.5 py-0.5",
+						line.type === "added" && "border-l-green-500 bg-green-500/10",
+						line.type === "removed" && "border-l-red-500 bg-red-500/10",
+						line.type === "context" && "border-l-transparent",
+					)}
+					key={`${i}-${line.type}`}
```

Biome (2.4.16)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)